scripts/docker-run.sh

```diff
@@ -8,5 +8,5 @@
 #  To build topology:
 #  ./scripts/docker-run.sh ./scripts/build-tools.sh
 
-docker run -i -v `pwd`:/home/sof/work/sof.git \
+docker run -i -t -v `pwd`:/home/sof/work/sof.git \
```

Comment:
@cujomalainey 
There is some issue with tty is not enabled in CI. Can you try to have some option passed in the scripts for manually usage?

Reply:
@xiulipan what issue? It should work, because what @cujomalainey added should be possible and should be default behaviour imho, not something that is enabled with flag.

Reply:
Hmm, I wonder if the bots are not providing everything docker needs to run with a tty. My recommendation would be to update the bot. Otherwise we should split off a second script imo.

Reply:
Considering travis handled the change fine. I would think jenkins is missing something.

Reply:
@xiulipan could this Jenkins failure be related to other CI issues atm. If this works on Travis then it should be good ?

Reply:
@lgirdwood @cujomalainey I am not sure if this is Jenkins issue.

`the input device is not a TTY`

Reply:
This is likely tied to the flag since the flag tells docker to allocate a tty, but imo jenkins should be handling this as some of the scripts (menuconfig for example) require that flag. There is likely something missing in the Jenkins config

Reply:
@cujomalainey Not sure what happened to the Jenkins scripts. I will try to workaround the issue by not using the scripts. Then we can merge this patch.

Reply:
@xiulipan Did you have a time to fix this?

Reply:
@jajanusz Can not fix with in Jenkins.

I will try to use a workaround for our Jenkins scripts. After I have update that I will approval and notify you to merge

Reply:
According to stackoverflow it is not possible to jenkins to handle that flag. Then I would recommend Jenkins pass in a flag to this script to disable the -t flag. I think jenkins should handle the exception case not the end user personally

Reply:
does the xcc jenkins build not use the container?

Reply:
@cujomalainey Yes, that was why I need to change it into no -t at first time. To make life easier I will replace the scripts with cmd directly. I will notify you when I done the change.
I am still working on to test it.

XCC do not use docker to contain, it is some binary release and we can not put them into docker.